exporter/sumologicexporter/exporter.go

```diff
@@ -146,7 +147,27 @@ func newTracesExporter(
 // start starts the exporter
 func (se *sumologicexporter) start(ctx context.Context, host component.Host) (err error) {
 	se.host = host
-	return se.configure(ctx)
+	err = se.configure(ctx)
```

Comment:
I think we need to recreate the sender inside `configure`. `configure` may be called asynchronously as a result of receiving an unauthorized error from the remote, in which case we want to get the data urls again. This used to be done implicitly by creating a new sender, but with this change we need to do so explicitly and handle any resultant concurrency issues.

Reply:
makes sense 💪